### ISSUE TITLE
Thank you very much for the program!

### ISSUE THREAD

**Issue by mrkaban** [frescobaldi/frescobaldi#1334]
Thank you very much for the program released under a free open source license! Added a program to a Russian-language site about free software, here is the [link](https://xn--90abhbolvbbfgb9aje4m.xn--p1ai/%D0%BC%D1%83%D0%BB%D1%8C%D1%82%D0%B8%D0%BC%D0%B5%D0%B4%D0%B8%D0%B0/%D1%81%D0%B8%D0%BD%D1%82%D0%B5%D0%B7/frescobaldi-%D1%80%D0%B5%D0%B4%D0%B0%D0%BA%D1%82%D0%BE%D1%80-%D0%BD%D0%BE%D1%82.html).

Please do not abandon the project, this is a very interesting program!

Will the releases be published here?

**Comment by wbsoft** [comment]
Yes, a 3.1.3 release is imminent. Thanks for your nice words! Although Frescobaldi is a free-time project for me (a professional musician), I will certainly not abandon it!
